win32/src/win32apimodule.cpp

```diff
@@ -5390,6 +5390,48 @@ static PyObject * PyLoadString(PyObject *self, PyObject *args)
 	return rc;
 }
 
+// @pymethod PyHANDLE|win32api|LoadResource|Finds a resource in a PE file. Returns a HRSRC(HANDLE) to the resource if found, otherwise returns None.
+static PyObject * PyFindResource(PyObject *self, PyObject *args) {
+    
+    PyObject *ret;
+    
+    PyObject *obhModule;
+    PyObject *obName;
+    PyObject *obType;
+    
+    HMODULE hModule;
+    LPTSTR lpName = NULL;
+    LPTSTR lpType = NULL;
+    
+    if ( !PyArg_ParseTuple(args, "OOO|H:FindResource",
+			&obhModule, // @pyparm <o PyHANDLE>|handle||The handle of the module containing the resource.  Use None for currrent process executable.
+			&obName,    // @pyparm <o PyResourceId>|name||Name of the resource to find
+			&obType     // @pyparm <o PyResourceId>|type||The type of resource to find.
+		))
+		return NULL;
+        
+        
+    if (PyWinObject_AsHANDLE(obhModule, (HANDLE *)&hModule)
+        &&PyWinObject_AsResourceId(obType, &lpType) 
+		&&PyWinObject_AsResourceId(obName, &lpName)) {
```

Comment:
slightly strange indentation here - it should match the line above. Please also add spaces after "&&"

Reply:
I was using spaces as indentation in my editor and your code was using tabs so the diff on git is weird. I converted my code to use tabs as per your code